frontend/src/components/BoardCreationForm/BoardCreationForm.jsx

```diff
@@ -28,9 +28,12 @@ const BoardCreationForm = () => {
       headers: { 'Content-Type': 'application/json' },
       body: JSON.stringify(newBoard),
     };
-    return fetch('/boards', requestOptions)
-      .then((response) => response.json())
-      .then((res) => navigate(`/board/${res.id}`));
+    return (
+      fetch('/boards/', requestOptions)
+        .then((response) => response.json())
+        // TODO obsluzyc mozliwosc blednej odpowiedzi
+        .then((res) => navigate(`/board/${res.id}`))
```

Comment:
Raczej .catch mażemy na końcu łańcucha :)))

```suggestion
        .then((res) => navigate(`/board/${res.id}`))
        // TODO obsluzyc mozliwosc blednej odpowiedzi
```